cupaloy.go

```diff
@@ -75,7 +77,11 @@ func (c *Config) snapshot(snapshotName string, i ...interface{}) error {
 
 	prevSnapshot, err := c.readSnapshot(snapshotName)
 	if os.IsNotExist(err) {
-		return c.updateSnapshot(snapshotName, snapshot)
+		if c.createNewAutomatically {
+			return c.updateSnapshot(snapshotName, snapshot)
+		}
+		//TODO: should an error still be printed here?
```

Comment:
@byang223 Are you expecting snapshot updates to occur totally silently when this option is enabled?